openupgrade_scripts/scripts/account/18.0.1.3/pre-migration.py

```diff
@@ -135,23 +134,6 @@ def adapt_account_move_sending_data(env):
             '{author_user_id}', sending_data::jsonb->'sp_user_id')
         WHERE sending_data IS NOT NULL AND sending_data::jsonb ? 'sp_user_id'""",
     )
-    # mail_template_id -> mail_template:
-    env.cr.execute(
-        """
-        SELECT id, sending_data::jsonb->'mail_template_id' as mail_template_id
-        FROM account_move
-        WHERE sending_data IS NOT NULL AND sending_data::jsonb ? 'mail_template_id'""",
-    )
-    for move_id, mail_template_id in env.cr.fetchall():
-        mail_template = env.ref(mail_template_id)
-        openupgrade.logged_query(
-            env.cr,
-            f"""
-            UPDATE account_move
-            SET sending_data = jsonb_set(sending_data::jsonb - 'mail_template_id',
-                '{{mail_template}}', {mail_template})
-            WHERE id = {move_id}""",
-        )
```

Comment:
the sending_data field is only used for the batch wizard, and there the mail template [can't be set at all](https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_send_batch_wizard.py#L86-L89)